include/tse.md

```diff
@@ -0,0 +1,57 @@
+Dieser Artikel erklärt den Umgang mit einer technischen Sicherheitseinrichtung (TSE) in Verbindung mit einem Gerät, auf dem pretixPOS läuft. 
+Jede Kasse im deutschen Rechtsraum muss mit einer TSE ausgestattet sein. 
+Wenn Sie eine Kasse von pretix oder von einem Drittanbieter gekauft haben, dann müssen Sie dort eine TSE einbauen. 
+Physikalisch handelt es sich bei einer TSE meistens um eine microSD-Karte oder um einen USB-Stick. 
+
+Wenn Sie eine Kasse für den Gebrauch mit pretixPOS **mieten**, dann mieten Sie damit auch eine TSE und Sie müssen sich nicht weiter darum kümmern. 
+
+## Voraussetzungen 
+
+Sie benötigen ein Kassengerät mit der App pretixPOS, eine TSE und eine stabile Internetverbindung. 
+
+## Anleitung 
+
+Die folgenden Abschnitte behandeln die ordnungsgemäße Einrichtung, Entfernung, und das Tauschen einer alten TSE gegen eine neue. 
+
+### TSE einrichten
+
+Setzen Sie die TSE in Ihr Kassengerät ein. 
+Schalten Sie die Kasse ein, öffnen Sie pretixPOS, tippen Sie den Button :btn:Einstellungen: und dann :btn-icon:fa3-gavel: Fiskalisierung / Technische Sicherheitseinrichtung:. 
+Tippen Sie den Button :btn-icon:fa3-microchip: Neue Sicherheitseinrichtung hinzufügen:. 
+Wählen Sie in dem Dialog den Typ (Hersteller) der TSE, die Sie soeben eingesetzt haben, und tippen Sie :btn:OK:. 
+
+Falls Ihre TSE noch nie zuvor benutzt wurde, dann wählen Sie eine 5-stellige PIN, eine 6-stellige PUK, sowie eine 5-stellige Zeitadmin-PIN. 
+Notieren Sie diese Daten unbedingt und verwahren Sie sie an einem sicheren Ort. 
+Sollten die Zugangsdaten zu einer TSE verloren gehen, dann kann sie nicht mehr benutzt werden. 
+
+!!! Note 
+    Verwahren Sie die Daten so, dass sie nicht in die Hände von Unbefugten gelangen können. 
+    Sorgen Sie aber auch dafür, dass die Daten auffindbar sind, wenn die Kasse genutzt werden soll, Sie aber nicht persönlich anwesend sind. 
+    Treffen Sie Vorkehrungen für den Fall, dass sich Ihre Zuständigkeit ändert (z.B. wenn Sie Ihren Job wechseln). 
+
+Falls Ihre TSE bereits benutzt wurde, dann geben Sie PIN, PUK und Zeitadmin-PIN ein. 
+
+In beiden Fällen stellen Sie sicher, dass Sie die richtigen Daten eingegeben haben und tippen dann :btn:OK:. 
+Es dauert einen Moment, die TSE einzurichten. 
+
+!!! Warnung 
+    Während pretixPOS die TSE einrichtet, dürfen Sie sie **auf keinen Fall** entfernen, die App beenden, oder die Kasse ausschalten. 
+    Dies kann dazu führen, dass die TSE unbrauchbar wird. 
+
+Sobald pretixPOS damit fertig ist, die TSE einzurichten, werden Sie auf die Seite mit dem Titel "Sicherheitseinrichtung verwalten" geleitet. 
```

Comment:
Für die weiteren Optionen in dem Menü könnte man auf das pretixPOS-Handbuch mit genauen Erklärungen zu jedem Eintrag verweisen.